content/en/docs/examples/binding and validation

```diff
@@ -0,0 +1,33 @@
+# This viminfo file was generated by Vim 8.0.
```

Comment:
missing file extension?

Reply:
@appleboy thanks! invalid file, I remove it!